data_transfer/main.py

```diff
@@ -1,99 +1,25 @@
-    # NOTE: simulates initiation of tasks upon metadata download
-    # TODO: in practice the tasks should be invoked within the batch job.
-    for record in records_not_downloaded(DeviceType.DRM):
-        # Each task should be idempotent. Returned values feeds subsequent task
-        mongoid = dreem_tasks.task_download_data(study_site, record.id)
-        mongoid = dreem_tasks.task_preprocess_data(mongoid)
-
-    # Data is finalised and moved to a folder in /uploading/
-    shared_jobs.prepare_data_folders()
-    # All said folders FOR ALL DEVICES are uploaded once per day
-    shared_jobs.batch_upload_data()
-
-
-def vttsma_dag() -> None:
-    """
-    Directed acyclic graph (DAG) representing data_transfer pipeline as used for all devices
-    Note that VTT does not distinguish between study sites
-
-    NOTE/TODO: this method simulates the pipeline.
-    """
-    vttsma_jobs.batch_metadata()
-
-    # NOTE: simulates initiation of tasks upon metadata download
-    # TODO: in practice the tasks should be invoked within the batch job.
-    for record in records_not_downloaded(DeviceType.SMA):
-        # Each task should be idempotent. Returned values feeds subsequent task
-        mongoid = vttsma_tasks.task_download_data(record.id)
-        mongoid = vttsma_tasks.task_preprocess_data(mongoid)
-
-    # Data is finalised and moved to a folder in /uploading/
-    shared_jobs.prepare_data_folders()
-    # All said folders FOR ALL DEVICES are uploaded once per day
-    shared_jobs.batch_upload_data()
-
-
-def byteflies_dag(study_site: StudySite) -> None:
-    """
-    Directed acyclic graph (DAG) representing dreem data pipeline:
-
-        batch_metadata
-            ->task_download_data
-            ->task_preprocess_data
-            ->task_prepare_data
-        ->batch_upload_data
-
-    NOTE/TODO: this method simulates the pipeline.
-    """
-
-    # TODO: ensure that we get the End of Day as intended based on automation schedule
-    # TODO: should we ensure overlap with previous run? How often do we run this DAG?
-    # TODO: perhaps pull this one abstraction higher (into the init of this DAG?)
-    data_period = get_period_by_days(datetime.today(), 2)  # NOTE: two days for testing
-    byteflies_jobs.batch_metadata(*data_period, study_site)
-
-    for record in records_not_downloaded(DeviceType.BTF):
-        # Each task should be idempotent. Returned values feeds subsequent task
-        mongoid = byteflies_tasks.task_download_data(record.id)
-        mongoid = byteflies_tasks.task_preprocess_data(mongoid)
-
-    # Data is finalised and moved to a folder in /uploading/
-    shared_jobs.prepare_data_folders(DeviceType.BTF)
-    # All said folders FOR ALL DEVICES are uploaded once per day
-    shared_jobs.batch_upload_data(DeviceType.BTF)
+from data_transfer.config import config
+from data_transfer.dags import btf, drm, sma
+from data_transfer.utils import DeviceType, StudySite
 
+fileConfig("logging.ini", disable_existing_loggers=False)
```

Comment:
Default is True, so can remove that parameter. It would be possible to have two logging streams and for to be output to a file. Perhaps something we can look into in a seperate PR?

Reply:
Separate PR - agreed!